third-party/gmp/Makefile

```diff
@@ -11,11 +11,24 @@ include $(CHPL_MAKE_HOME)/make/Makefile.base
 GMP_UNPACKED_DIR = gmp-$(GMP_VERSION)
 GMP_TARBALL = $(GMP_UNPACKED_DIR)a.tar.bz2
 
+#
+# Cray X* builds are cross-compilations, as are Intel Phi KNC builds.
+#
+GMP_CROSS_COMPILED=no
+ifneq (, $(filter cray-x%,$(CHPL_MAKE_TARGET_PLATFORM)))
+CHPL_GMP_CFG_OPTIONS += --build=x86_64-unknown-linux-gnu \
+                        --host=x86_64-cnl-linux-gnu
```

Comment:
Yes, but I should probably change that to "cle" for Compute Linux Environment, which is the modern name. It seems to work just as well, because that part of the arch quad doesn't matter. I just tried it on hwloc.